idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -708,8 +718,7 @@ def run_rapsearch2_remotely(input_fasta, lazy_run):
     remote_index_dir = "%s/references/nr_rapsearch" % remote_home_dir
     # split file:
     chunk_nlines = 2*RAPSEARCH_CHUNK_SIZE
-    part_suffix = "-chunksize-%d-part-" % RAPSEARCH_CHUNK_SIZE
```

Comment:
part_suffix is also used in run_rapsearch2_remotely

Reply:
oh sorry

Reply:
No worries :)